tests/integration/test_sqs.py

```diff
@@ -732,6 +732,40 @@ def test_fifo_queue_send_multiple_messages_single_receive(self):
 
     def test_fifo_queue_send_multiple_messages_multiple_receives(self):
         queue_url, number_of_messages, results = self._run_test_fifo_queue_send_multiple_messages()
+        first_receives = number_of_messages // 2
+
+        # receive multiple message first time
+        messages = self.client.receive_message(
+            QueueUrl=queue_url, MessageAttributeNames=['All'], MaxNumberOfMessages=first_receives
+        )
+
+        # asset the received messages data first time
+        self.assertEqual(first_receives, len(messages['Messages']))
+        for i in range(first_receives):
+            self.assertEqual('message-{}'.format(i), messages['Messages'][i]['Body'])
+            self.assertEqual(results[i]['MD5OfMessageBody'], messages['Messages'][i]['MD5OfBody'])
+            self.assertEqual(results[i]['MessageId'], messages['Messages'][i]['MessageId'])
+
+            # delete message to receive next message in queue
+            self.client.delete_message(QueueUrl=queue_url, ReceiptHandle=messages['Messages'][i]['ReceiptHandle'])
+
+        second_receives = number_of_messages - first_receives
+
+        # receive multiple message second time
+        messages = self.client.receive_message(
+            QueueUrl=queue_url, MessageAttributeNames=['All'], MaxNumberOfMessages=second_receives
```

Comment:
Wondering if we should extend this test to also include a `receive_message` call without specifying `MaxNumberOfMessages` at all? (as this seems to be the issue reported in https://github.com/localstack/localstack/pull/2713/files#r455313289 ..)